test/Concurrency/actor_isolation.swift

```diff
@@ -834,31 +834,29 @@ actor LazyActor {
     lazy var l25: Int = { [unowned self] in self.l }()
 
     nonisolated lazy var l31: Int = { v }()
-    // expected-warning@-1 {{'nonisolated' is not supported on lazy properties; this is an error in the Swift 6 language mode}}
+    // expected-error@-1 {{'nonisolated' cannot be applied to mutable stored properties}}
```

Comment:
I think we need to downgrade the diagnostic to a warning until Swift 6 if it's a lazy var or the var has an attached property wrapper.